docs/02-tutorial.mdx

```diff
@@ -0,0 +1,281 @@
+
+Using the Privacy Computation Engine goes into three steps:
+
+1. **[App Management](/docs/access-management/set-up)**: register and obtain an application id;
+2. **[Configuration](./computation/configuration)**: configure the service using its configuration API;
+3. **Development**: call the Privacy Computation Engine from your registered application to handle Privacy Requests
+
+Step _(1) App Management_ and _(2) Configuration_ don't need to be addressed for this tutorial.
+
+:::tip Pre-configured Instance
+
+To make things easier for this tutorial, we have already pre-configured a dedicated custom instance for you to play with:
+
+<div class="text--center">
+
+**`https://devkit-pce-staging.azurewebsites.net/v0/`**
+
+</div>
+
+:::
+
+Step _(3) Development_ can be achieved directly by calling the standalone Privacy Computation Engine [REST API](https://devkit-pce-staging.azurewebsites.net/swagger/).
+
+However, the true strength of the blindnet devkit comes when combining its various components, thus drastically improving your developer experience and productivity while giving you access to additional features.
+
+Starting with our premade user interfaces.
+
+<!-- ## 2. Capture Data -->
+<!-- TODO: add information on how to capture data without the Capture Componet, i.e. call the demo storage service API to save a data from a simple HTML form -->
+
+## 2. Add Privacy-Rights Management Capabilities to Your Application
+
+User interfaces for the blindnet devkit (aka _[Privacy Components for the Web](https://github.com/blindnet-io/privacy-components-web)_) are delivered as standard Web Components available on both npm and a CDN.
+
+As a consequence, installing the blindnet devkit web interfaces doesn't involve any specificity, as you can install and use them as you would with any other standard Web Component.
+
+:::info
+
+See [Using Web Components](/docs/interfaces/using-web-components) for more information on how to integrate Web Components in your existing project.
+
+:::
+
+In the interest of simplicity and clarity, let's see how to integrate these components into a simple static HTML page, starting with the Privacy Request Capture Interface (PRCI).
+
+PRCI is a simple interface allowing your users to assert their privacy rights by sending you a simple yet standardized [Privacy Request](/docs/references/lexicon#privacy-request). It is an entry point for your user to privacy, not only allowing you to cover GDPR and other privacy-related legislations but also and more importantly, to build trust by showing your users their privacy rights are tangibly handled, enforced and respected.
+
+To integrate this interface into your web page, you'll first need to load the associated bundle (included in the `@blindnet/prci` npm package) from the unpkg CDN by adding a `<script>` tag to your HTML document:
```

Comment:
```suggestion
To integrate this interface into your web page, you can load the associated bundle (included in the `@blindnet/prci` npm package) from the unpkg CDN by adding a `<script>` tag to your HTML document:
```
Invitational language instead of a conditional

Reply:
I'd totally agree with you in a more generic context. But this is a tutorial, so it's still better to formulate it in a directive way ("do this, do that"), which means invitational language should be avoided.